apps/serialization/gateway.proto

```diff
@@ -347,6 +354,25 @@ syntax = "proto3";
     Unit unit = 2;
   }
 
+  message Unlock {
+    string name = 1;
+    Upgrade upgrade = 2;
+  }
+
+  message Upgrade {
+    string name = 1;
+    string description = 2;
+    int32 group = 3;
+    repeated CurrencyCost cost = 4;
+    repeated Buff buffs = 9;
```

Comment:
Is there a reason for this jump, or could we assign number 5 to `buffs`?

Reply:
Not really, probably comes from fields I added initially & removed afterwards